content/actions/creating-actions/publishing-actions-in-github-marketplace.md

```diff
@@ -64,6 +64,16 @@ To remove a published action from {% data variables.product.prodname_marketplace
 1. Select **Publish this action to the {% data variables.product.prodname_marketplace %}** to remove the check from the box.
 1. Click **Update release** at the bottom of the page.
 
+## Transferring an action repository
+
+- You can transfer an action repository to another user or organization. For more information, see "[AUTOTITLE](/repositories/creating-and-managing-repositories/transferring-a-repository)."
+
+- When a repository admin transfers an action repository, {% data variables.product.prodname_dotcom %} automatically creates a redirect from the previous URL to the new URL, meaning workflows that use the affected action do not need to be updated.
+
+- Actions published on {% data variables.product.prodname_marketplace %} are linked to a repository by their unique `name` identifier, meaning you can publish new releases of an action from the transferred repository under the same [{% data variables.product.prodname_marketplace %}] listing. If an action repository is deleted, the {% data variables.product.prodname_marketplace %} listing is also deleted, and the unique `name` identifier becomes available.
```

Comment:
```suggestion
You can transfer an action repository to another user or organization. For more information, see "[AUTOTITLE](/repositories/creating-and-managing-repositories/transferring-a-repository)."

When a repository admin transfers an action repository, {% data variables.product.prodname_dotcom %} automatically creates a redirect from the previous URL to the new URL, meaning workflows that use the affected action do not need to be updated.

Actions published on {% data variables.product.prodname_marketplace %} are linked to a repository by their unique `name` identifier, meaning you can publish new releases of an action from the transferred repository under the same {% data variables.product.prodname_marketplace %} listing. If an action repository is deleted, the {% data variables.product.prodname_marketplace %} listing is also deleted, and the unique `name` identifier becomes available.
```

We can delete the bullets here—since these are distinct pieces of information rather than (for example) a list of options or substeps related to a specific action, this makes more sense as separate paragraphs 🙂

I also removed the brackets around one of the instances of `{% data variables.product.prodname_marketplace %}`. We only need the curly braces for the variable to render properly, and any other set of parentheses will appear in the docs themselves.